composer.json

```
@@ -3,7 +3,7 @@
     "type": "composer-plugin",
     "license": "OSL-3.0",
     "require": {
-        "php": "^7.2",
+        "php": "^7.2||^8.1",
```

```suggestion
        "php": "~8.1.0||~8.2.0",
```
Should we update the php requirements to match current Adobe Commerce develop requirements and allow using PHP 8 features?

Can you please update the PHP versions used for tests as well? (in https://github.com/magento/magento-vcs-installer/blob/develop/.github/workflows/ci.yaml)